packages/site/src/components/RenderComponentShowCode.tsx

```diff
@@ -42,11 +42,9 @@ export const RenderComponentShowCode = ({ children }: PropsWithChildren) => {
         </div>
       </Box>
       {codeVisible && (
-        <Box background="base-blue--800" padding={"small"} radius="base">
-          <pre>
-            <code className="language-javascript">{code}</code>
-          </pre>
-        </Box>
+        <pre>
```

Comment:
Since `pre` is doing all of this now, don't need the Box

Open to the idea that pre should be less opinionated and we do more configuration on a per-usage instance though